package.json

```diff
@@ -15,7 +17,7 @@
     "react-split-pane": "^0.1.74",
     "redux": "^3.7.2",
     "redux-devtools-extension": "^2.13.2",
-    "redux-little-router": "^14.2.3",
+    "redux-little-router": "^15.0.0",
```

Comment:
could you describe what was the problem?

Reply:
While trying to make the project buildable it was needed at some point (it was somehow related to https://github.com/FormidableLabs/redux-little-router/pull/266), but it is not longer needed (idk why), so I'll rollback it